### PR TITLE
services: libs: messages: First commit

### PR DESCRIPTION
**Pull request by patrickelectric**
Signed-off-by: Patrick José Pereira <patrickelectric@gmail.com>

### CODE REVIEW

File: services/libs/messages.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+            ('max_abs_pitch', 'B', '360degree/256'),
+            ('battery_voltage', 'B', 'decavoltage/256'),
+            ('battery_current', 'B', '64amps/256'),
+            ('solar_voltage', 'B', '64volts/256'),
+            ('solar_power', 'B', '200w/256'),
+            ('throttle_first', 'B', '100%/256'),
+            ('throttle_second', 'B', '100%/256'),
+            ('air_temperature', 'B', '64celsius/256'),
+            ('water_temperature', 'B', '64celsius/256'),
+            ('cpu', 'B', '100/256'),
```
Comment (joaoantoniocardoso) [inline services/libs/messages.py:23]: Just to note that CPU is usually measured from 0 to 100% by core, meaning that Raspberry would scale from 0 to 400%, so to represent from 0 to 100 we would need to convert it.

File: services/libs/messages.py [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+            ('wind_speed', 'B', '64meters/second'),
+            ('wind_angle', 'B', '360degree/256'),
+            ('gps_fix_type', 'B', 'no gps/no fix/2d fix/3d fix'),
+            ('sat_number', 'B', 'number'),
+            ('lattitude', 'f', 'degrees'),
+            ('longitude', 'f', 'degrees'),
+            ('next_waypoint_lattitude', 'f', 'degrees'),
+            ('next_waypoint_longitude', 'f', 'degrees'),
+            ('vdop', 'H', 'uin16_t'),
+            ('hdop', 'H', 'uin16_t'),
```
Comment (joaoantoniocardoso) [inline services/libs/messages.py:38]: what are these two?

Comment (patrickelectric) [reply services/libs/messages.py:38]: For GPS imprecision: https://en.wikipedia.org/wiki/Dilution_of_precision_(navigation)

File: services/libs/messages.py (hunk 1 above)
Comment (patrickelectric) [reply services/libs/messages.py:23]: Yes

**Comment by patrickelectric** [comment]
@Williangalvani

**Pull request merged**